# Don't pull in another quatro version than the sources in easyconfig.
# Author: Lara Peeters (Ghent University)
diff -ruN rstudio-2023.09.1-494.orig/dependencies/common/install-panmirror rstudio-2023.09.1-494/dependencies/common/install-panmirror
--- rstudio-2023.09.1-494.orig/dependencies/common/install-panmirror	2023-10-13 16:52:00.000000000 +0200
+++ rstudio-2023.09.1-494/dependencies/common/install-panmirror	2026-08-06 17:55:18.088053040 +0200
@@ -41,11 +41,11 @@
   echo "quarto repo already cloned in '$QUARTO_DIR'"
 
   pushd $QUARTO_DIR
-  git fetch
-  git reset --hard && git clean -dfx
-  git checkout main
+  # git fetch
+  # git reset --hard && git clean -dfx
+  # git checkout main
   # git checkout release/rstudio-mountain-hydrangea
-  git pull
-  git rev-parse HEAD
+  # git pull
+  # git rev-parse HEAD
   popd
 fi
